### PR TITLE
Feature: Update upload_to file to accomodate new filemanager

### PR DESCRIPTION
**Pull request by ayu023ban**

### CODE REVIEW

File: utils/upload_to.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import mimetypes
 import os
 import uuid
-
```
Comment (dhruvkb) [inline utils/upload_to.py:4]: Why? Please keep newlines between imports of different kinds.

File: utils/upload_to.py [hunk 2]
```diff
@@ -33,39 +33,56 @@ def __call__(self, instance, filename):
...
-                destination,
-            )
-            os.remove(path)
-        except FileNotFoundError:
-            pass
-
         return destination
+
+    def get_file_name(self, path, filename):
+        """Return Updated file name in case of file already exists
```
Comment (dhruvkb) [inline utils/upload_to.py:75]: See other comments to see how we style comments.

File: utils/upload_to.py [hunk 3]
```diff
@@ -33,39 +33,56 @@ def __call__(self, instance, filename):
...
-            pass
-
         return destination
+
+    def get_file_name(self, path, filename):
+        """Return Updated file name in case of file already exists 
+        :param path: path of folder in which the file will be stored 
+        :param filename: the original name of the file
+        :return: the path to the uploaded image
+        """
```
Comment (dhruvkb) [inline utils/upload_to.py:79]: Leave a newline after the comments.

File: utils/upload_to.py [hunk 4]
```diff
@@ -33,39 +33,56 @@ def __call__(self, instance, filename):
         :param filename: the original name of the file, used for the extension
         :return: the path to the uploaded image
         """
-
```
Comment (dhruvkb) [inline utils/upload_to.py:36]: This is required, please do not remove.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Please see some older files from the repo to see the conventions and style guide we follow. It's unfortunate that you have to infer this from existing files instead of having an actual document, but maybe you can take it up.

**Comment by ayu023ban** [comment]
@dhruvkb  i have made the required changes.
please look into the latest patch once again

**Comment by pradumangoyal** [comment]
This custom `upload_to` should go in the respective app only.

**Pull request closed without merging**